### PR TITLE
Prepare for release v1.13.0

### PR DESCRIPTION
**Pull request by aanm**
See CHANGELOG.md for more details

Signed-off-by: André Martins <andre@cilium.io>

### CODE REVIEW

**Review by christarazi — APPROVED** [review]
:ship:

**Comment by coopreme** [comment]
🌮🌮🌮

**Review by nbusseneau — APPROVED** [review]

File: .github/workflows/build-images-releases.yaml [hunk 1]
```diff
@@ -94,6 +94,8 @@ jobs:
           tags: |
             ${{ github.repository_owner }}/${{ matrix.name }}:${{ steps.tag.outputs.tag }}
             quay.io/${{ env.QUAY_ORGANIZATION }}/${{ matrix.name }}:${{ steps.tag.outputs.tag }}
+            ${{ github.repository_owner }}/${{ matrix.name }}:stable
+            quay.io/${{ env.QUAY_ORGANIZATION }}/${{ matrix.name }}:stable
```
Comment (joestringer) [inline .github/workflows/build-images-releases.yaml:98]: Do we need to delete these lines from the corresponding file in v1.12 branch?

Comment (joestringer) [reply .github/workflows/build-images-releases.yaml:98]: @aanm

**Pull request merged**